AI/VCAI/VCAI.h

```diff
@@ -138,7 +141,10 @@ class VCAI : public CAdventureAI
 
 	friend class FuzzyHelper;
 
+	std::map<TeleportChannelID, shared_ptr<TeleportChannel> > knownTeleportChannels;
 	std::map<const CGObjectInstance *, const CGObjectInstance *> knownSubterraneanGates;
+	ObjectInstanceID destinationTeleport;

```

Comment:
>Or valid game save is possible during AI turn?

Currently it is impossible
Initialization in constructor is not enough, it may need reinit before each movement.